### ISSUE TITLE
Display 1 regular (no regression) chart and regression chart on same web page

### ISSUE THREAD

**Issue by paulmillard831** [streamlinesocial/highcharts-regression#83]
Hi, I can get the regression charts to work.  In some web pages, I have both regular charts that I do not want regression line displayed.  How can I do this?

BTW, Great work!

Thank you,
Paul

**Comment by phpepe** [comment]
I guess you mean 'series' instead of charts. If thats the case then on the last link in the documentation there is an example:  [Regression enabled per series](http://jsfiddle.net/phpepe/ajp5mobc/)
Basically add the "regression" settings on any series you need

**Comment by paulmillard831** [comment]
Perfect!  I am displaying two charts on same page.  One chart with no regression, while other has regression.  So to disable regression in first chart and prevent display errors, I ended up adding this to the series.  

series: [{
        regression: false
      	}]

**Comment by phpepe** [comment]
I was not aware of issues related multiple charts.
Do you have  jsFiddle example showing this error ?

**Comment by paulmillard831** [comment]
In most of all my charts I use the loading data with CSV.  https://www.highcharts.com/docs/working-with-data/data-module It works well as I call a function to which type of chart I'd like to see displayed with parameters passed to a chart function.  

The other chart function I created was the scatter regression using your code.  Once I used the included highcharts-regression.js I was having problems with my original CSV loaded charts being loaded.  Once I put in regression: false in the series they work.  Right now my scatter regression is using JSON and pushing data into array.

On a side note, how can I use regression using data loaded using CSV?  Right now my charts work using JSON.  My preference is still to use the CSV loaded data.  I'm guessing I'll need to figure out how to take the CSV data and put into object array (points) and use data: points in the series.  Not sure how to do that at the moment.  Thanks for your help!
